### PR TITLE
[sd-2234]-(publish)Distinguishable confirmation for publish and kill(…

### PR DESCRIPTION
**Pull request by sjunaid**
…improvement)

### CODE REVIEW

**Comment by amagdas** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Comment by plamut** [comment]
Unit test missing, otherwise good.

File: client/app/scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -450,7 +450,7 @@
          */
         this.confirmPublish = function confirmPublish(action) {
             return modal.confirm(
-                gettext('There are some unsaved changes, do you want to save it and ' + action + ' now?'),
+                $interpolate(gettext('There are some unsaved changes, do you want to save it and {{ action }} now?'))({action: action}),
                 gettext('Save changes?'),
                 gettext('Save and ' + action),
```
Comment (petrjasek) [inline client/app/scripts/superdesk-authoring/authoring.js]: here you can use it too..

**Pull request merged**